sdk/dotnet/Pulumi.FSharp/Library.fs

```diff
@@ -1,15 +1,113 @@
 ﻿namespace Pulumi.FSharp
 
-open System.Collections.Generic
 open Pulumi
 
 [<AutoOpen>]
 module Ops =
-    let input<'a> (v: 'a): Input<'a> = Input.op_Implicit v
+
+    /// <summary>
+    /// Wraps a raw value into an <see cref="Input{'a}" />.
+    /// </summary>
+    let input<'a> (value: 'a): Input<'a> = Input.op_Implicit value
+
+    /// <summary>
+    /// Wraps an <see cref="Output" /> value into an <see cref="Input{'a}}" /> value.
+    /// </summary>
     let io<'a> (v: Output<'a>): Input<'a> = Input.op_Implicit v
 
+    /// <summary>
+    /// Wraps a collection of items into an <see cref="InputList{'a}}" />.
+    /// </summary>
+    let inputList<'a> (items: seq<Input<'a>>) =
+        let result = new InputList<'a> ()
```

Comment:
Nit: Is there a reason why you put a space between `InputList<'a>` and `()`? IIRC, most code I've seen doesn't put any space between a type and its constructor args.

Reply:
Old habit... will remove